docs/how-to-use-and-FAQ/use-ncnn-with-pytorch-or-onnx.md

```diff
@@ -22,9 +128,10 @@ x = torch.rand(1, 3, 224, 224)
 # Export the model
 torch_out = torch.onnx._export(model, x, "resnet18.onnx", export_params=True)
 ```
+</details>
 
 ## simplify onnx model
```

Comment:
```suggestion
## simplify onnx model (deprecated)
```

Reply:
this suggestion was successfully applied.